tests/nlu/utils/test_hf_transformers.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import numpy as np
+
+from rasa.nlu.utils.hugging_face.hf_transformers import HFTransformersNLP
+from rasa.nlu.training_data import Message
+
+
+@pytest.mark.parametrize(
+    "input_sequence_length, model_name, should_overflow",
+    [(20, "bert", False), (1000, "bert", True), (1000, "xlnet", False)],
+)
+def test_sequence_length_overflow_train(
```

Comment:
Can you please add types.

Reply:
also for the other tests

Reply:
I never saw types added to the tests and I think that's because the API should not be tested against specific types. The test should break if the input the API is wrong. Happy to add the types if that's not the case.

Reply:
part of our coding guidelines: https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e

Reply:
Thanks 👍  i'll add them